forge-core/src/main/java/forge/card/CardType.java

```diff
@@ -988,6 +988,8 @@ public static boolean isAPlanarType(final String cardType) {
      * Otherwise, simply return the input.
      * @param type a String.
      * @return the corresponding type.
+     *
+     * @deprecated
      */
     public static final String getSingularType(final String type) {
```

Comment:
imo this can be removed?

Reply:
normally you only want Singular to Plural for Gui stuff

I don't know of cases where you might want Plural to Singular